index.js

```diff
@@ -14,6 +14,11 @@ module.exports = function (input, length) {
 		return input;
 	}
 
+	var strippedTrailingSlash = input.replace(/\/$/, '');
+	if (strippedTrailingSlash.length === length) {
+		return strippedTrailingSlash;
+	}
```

Comment:
we can return complete url for this use case: `http://example.com/a/b/c/` -> `http://example.com/a/b/c`